tools/acrnlog/README.rst

```diff
@@ -1,63 +1,88 @@
 
-USAGE
-#####
-The ``acrnlog`` tool is launched as a service at boot, with 4 1MB log files limited.
-To change the log file limitation:
+Log files are saved in ``/tmp/acrnlog/``, so the log files would be lost
+after a system reset.
 
-- temporary change:
-    Stop the ``acrnlog`` service:
+Usage
+*****
 
- ::
+The ``acrnlog`` tool is launched as a service at boot, and limited to
+supporting four 1MB log files by default.  You can change this log file
+limitation temporarily or permanently.
 
- # systemctl disable acrnlog
 
-Restart ``acrnlog`` running in backgroud with size and number of files.
- For example:
+Temporary log file changes
+==========================
 
- ::
+You can temporarily change the log file setting by following these
+steps:
 
- # acrnlog -n 8 -s 4 &
+1. Stop the ``acrnlog`` service:
 
-Use ``get_loglevel``/``set_loglevel`` to query and change the hypervisor loglevel.
+   .. code-block:: none
 
-The ``mem_loglevel`` controls log to be saved using ``acrnlog``, while
-``console_loglevel`` controls log to output to console. For example:
+      # systemctl disable acrnlog
 
- ::
+2. Restart ``acrnlog``, running in the background, and specify the new
+   number of log files and their size (in MB).  For example:
 
-  ACRN:\>get_loglevel
-  console_loglevel: 2, mem_loglevel: 4
-  ACRN:\>set_loglevel 2 5
-  ACRN:\>get_loglevel
-  console_loglevel: 2, mem_loglevel: 5
+   .. code-block:: none
 
-- permanent change:
-   Edit ``/usr/lib/systemd/system/acrnlog.service`` to attached the ``-n`` and ``-s`` options to the ``ExecStart`` cmd, and restart the service. For example:
+      # acrnlog -n 8 -s 4 &
 
- ::
+You can use ``get_loglevel`` and ``set_loglevel`` to query and change
+the hypervisor loglevel.
 
-  ExecStart=/usr/bin/acrnlog -n 8 -s 4
+The ``mem_loglevel`` command controls the log to be saved using
+``acrnlog``, while the ``console_loglevel`` command controls the log
+output to the console. For example:
 
+.. code-block:: none
 
-BUILD & INSTALL
-##################
+   ACRN:\>get_loglevel
+   console_loglevel: 2, mem_loglevel: 4
+   ACRN:\>set_loglevel 2 5
+   ACRN:\>get_loglevel
+   console_loglevel: 2, mem_loglevel: 5
 
-::
+Permanent log file changes
+==========================
 
- # make
- 
-copy acrnlog to ``/usr/bin/`` and copy ``acrnlog.service`` to ``/usr/lib/systemd/system/``
+You can also permanently change the log file settings by
+editing ``/usr/lib/systemd/system/acrnlog.service`` and use the ``-n``
+and ``-s`` options on the ``ExecStart`` cmd, and restart the service.
+For example:
+
+.. code-block:: none
+
+   ExecStart=/usr/bin/acrnlog -n 8 -s 4
+
+
+Build and Install
+*****************
+
+Source code for the ``acrnlog`` tools is in the ``tools/acrnlog``
+folder.  Build the tools from source using:
+
+.. code-block:: none
+
+   # make
+
+and install (with your edited ``acrnlog.service`` file) using:
+
+.. code-block:: none
+
+   # cp acrnlog /usr/bin/
```

Comment:
`make install` will do that. The unfortunate thing is that `make install` does not install the systemd service file (which is the next line)... it would be better if it did so we can replace these two lines with `make install`

Reply:
Until the Makefile is updated, these instructions are correct then, right?